src/constants/index.ts

```diff
@@ -8,7 +8,7 @@ export const CHECKBOX_TYPE_STAR = "star";
 export type CHECKBOX_TYPE_STAR = typeof CHECKBOX_TYPE_STAR;
 export const TOOLTIP_COLOR = "#3B3649";
 export const TOOLTIP_DELAY = 1; // in seconds
-export const NAV_BAR_TOOLTIP_OFFSET = [0, -8];
+export const NAV_BAR_TOOLTIP_OFFSET = [0, 10];
```

Comment:
this seems like a big difference.. but I went back and read your description of why.